conftest.py

```diff
@@ -6,6 +6,11 @@
 import uuid
 
 import pytest
+
+# Setup ocean_lib.enforce_typing_shim before importing anything that uses it
+from ocean_lib.enforce_typing_shim import setup_enforce_typing_shim
```

Comment:
I think we might still have isort on for this repo, although it's possible we'll remove it soon. But until then, can you please add instructions to ignore isort for these lines? i.e. isort will try to make all imports happen between all instructions, unless you say otherwise..

Reply:
Unfortunately, I ended up ignoring the entire file.  I couldn't prevent isort from moving `setup_enforce_typing_shim()` down below the imports.  I tried a variety of combinations of `# isort: on/off/skip`.  I also played with `# fmt: on/off/skip`